src/c_api.rs

```diff
@@ -435,6 +451,25 @@ macro_rules! redis_json_module_export_shared_api {
             isJSON: JSONAPI_isJSON,
         };
 
+        static JSONAPI_V2: RedisJSONAPI_V2 = RedisJSONAPI_V2 {
```

Comment:
Why do we need to different structs here?  We only extended the struct so we can use the same struct for both.